.github/workflows/main.yml

```
@@ -1,44 +1,47 @@
-      - name: 🚀 Build iOS App with EAS
-        run: |
-          eas build --non-interactive --platform ios --profile production
+    build:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                profile: ["staging", "production"]
+
+        steps:
+            - name: 🏗️ Checkout repository
+              uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)